### PR TITLE
added express functionality

### PR DESCRIPTION
**Pull request by MaxaKozin**

### CODE REVIEW

File: contacts.js [hunk 1]
```diff
@@ -1,59 +1,74 @@
 const fs = require("fs").promises;
 const path = require("path");
+const shortid = require("shortid");
 
 contactsPath = path.resolve(__dirname, "db", "contacts.json");
```
Comment (odykyi) [inline contacts.js]: make contactsPath as const

File: contacts.js [hunk 2]
```diff
@@ -1,59 +1,74 @@
...
 
 async function listContacts() {
-  fs.readFile(contactsPath, "utf-8")
-    .then((contacts) => console.table(JSON.parse(contacts)))
-    .catch((err) => console.log(err.message));
+  return (contacts = await fs.readFile(contactsPath, "utf-8"));
```
Comment (odykyi) [inline contacts.js]: return await fs.readFile(contactsPath, "utf-8");

**Review by odykyi — CHANGES_REQUESTED** [review]
comments

**Comment by odykyi** [comment]
@MaxaKozin wait for changes

File: contacts.js (hunk 2 above)
Comment (MaxaKozin) [reply contacts.js]: Corrections done

File: contacts.js (hunk 1 above)
Comment (MaxaKozin) [reply contacts.js]: corrections done

**Review by odykyi — APPROVED** [review]

**Pull request merged**